catalog/transformer/catalog-transformer-xml/src/test/java/ddf/catalog/transform/xml/TestEscapingPrintWriter.java

```diff
@@ -87,48 +87,6 @@
   }
 
   @Test
```

Comment:
❓ No new test for this?

Reply:
I'm not going to add tests for the Apache commons escaping utility -- this class is merely a proxy for it now.

Reply:
Why did these tests need to be removed? Do they fail against the Apache commons utility?

Reply:
Correct. They are actually asserting things that are invalid according to the XML spec.